/**:
  ros__parameters:
    offset_x: 0.0
    offset_y: 0.0
    offset_z: 0.0
